### PR TITLE
issue #387: wabt-sys build error - updated wabt version in test_utils

### PR DESCRIPTION
**Pull request by murrayjbrown**
Updated version of wabt crate dependency specified in _test_utils/Cargo.toml_ to match that in _core/Cargo.toml_ to resolve error encounted compiling _wabt-sys._

### CODE REVIEW

**Comment by murrayjbrown** [comment]
Latest commit d7daba1 related to issue  #52 - refactored `hash_table::entry::Entry` and related referencing code to use a Box smart pointer for immutable content so that entry cloning will be more efficient for large content strings.

File: test_utils/Cargo.toml [hunk 1]
```diff
@@ -7,5 +7,5 @@ authors = ["Nicolas Luck <nicolas@lucksus.org>"]
 holochain_dna = { path = "../dna" }
 holochain_core = { path = "../core" }
 holochain_agent = { path = "../agent" }
-wabt = "0.2.0"
+wabt = "0.4"
```
Comment (thedavidmeister) [inline test_utils/Cargo.toml]: @murrayjbrown this was already approved by the team and could be merged, but the rest of the changes in this PR need to be reviewed separately

could you please split this PR into two as the scope of entry and wabt changes seems unrelated to each other?

Comment (murrayjbrown) [reply test_utils/Cargo.toml]: @thedavidmeister  Yeah, sorry about that. Will do.
